src/main/java/frc/robot/Constants.java

```diff
@@ -17,10 +17,32 @@
  */
 public final class Constants {
 
+  public static class ControlConstants
+  {
+    public static final int JOYSTICK_RIGHT_NUMBER = 0;
+    public static final int BUTTON_9 = 9;
+    public static final int BUTTON_10 = 10;
+    public static final int BUTTON_11 = 11;
+    public static final int BUTTON_12 = 12;
+  }
```

Comment:
these should go into the already-existing `OperatorConstants` inner class

button constants should be named following a certain naming scheme: which controller they're for and what they do. for example, `OP_LIFT_BTN` could be the button id for a command that lifts the arm; `DRIVER_TOGGLE_LL_BTN` could be a button id for a command that toggles the limelight pipeline